widgetssdk/src/main/java/com/glia/widgets/view/head/controller/ApplicationChatHeadLayoutController.kt

```diff
@@ -65,6 +68,12 @@ internal class ApplicationChatHeadLayoutController(
         }
         visitorMediaUseCase.onHoldState.unSafeSubscribe(::onHoldChanged)
         currentOperatorUseCase().unSafeSubscribe(::operatorDataLoaded)
+        screenSharingUseCase().unSafeSubscribe {
```

Comment:
Why do you `.filter { isCallVisualizerScreenSharingUseCase() }` in `ServiceChatHeadController.kt` but don't filter here, in `ApplicationChatHeadLayoutController.kt`?

Reply:
Fixed! Thank you for bringing it to my attention.